lib/count_users.py

```diff
@@ -106,7 +106,7 @@ def count_users(diagnosis_key_list, multiplier=10, auto_multiplier_detect=False,
         for multiplier_candidate in reversed(range(1, multiplier+1)):
```

Comment:
@micb25 thank you very much for the PR. 
Initially I thought that setting -m 5 (or -m 9) and -a at the same time would achieve what you need, because of line 106. 
Does that not work properly?
For that use case, my original naming / help text of the parameters is somewhat confusing, though, and should be improved.

Reply:
Indeed, it does work correctly with the additional command line parameter `-m 9`. I wasn't aware that the manually specified padding multiplier would affect the automatic multiplier detection. As a consequence, this makes my PR redundant.